content/Parents_content.php

```diff
@@ -37,3 +37,20 @@
 		</p>
 	</div>
 </div>	
+<div class="post">
+	<h1 class="title"><a>New Year Student Signups</a></h1>
+		</p>
+	</div>
+</div>
+<div class="post">
+	<h1 class="title"><a>Want to join for our team?</a></h1>
+	<p class="byline"><small>Posted on August 24th, 2015</small></p>
+	<div class="entry">
+		<p>
+Please fill out the below papers and bring them with you to the next open shop night: </br>
+			<ul></br><a style=\"target-new: tab\" target=\"new\" href="toolcert.docx">FRC Tool Certification (docx)</a></ul></br>
```

Comment:
Why the `<ul>` here?
`<ul>` should include `<li>` as their children and wrap your anchors...

```html
<ul class="papers">
  <li class="papers__item"><a target="_blank" href="some-url">Some Text</a></li>
</ul>
```

```css
.papers {
  padding: 0;
}

.papers__item: {
  padding: 16px 0; // should use rem, but that is a long discussion 
}
```
Some things to notice:
* The use of `target-new: xxx` is a big 👃, `target="_blank"` allows for the user specified setting is more supported across browsers.